### DISCUSSION TITLE
School does not providing email address to students.

### DISCUSSION THREAD

**Discussion by eensungkim** [community/community#53829] [GitHub Education]
### Hi everyone,

Hello, I am working as an Education Operation Manager at "Codestates".
We do not provide email addresses to our students. Students must register with their own email addresses.
However, when students choose "Codestates" to get benefits for students, they are required to use school-issued email address and cannot proceed to the next step.
How can we solve this issue?

**Comment by samus-aran** [comment]
Hi @eensungkim  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍

**Reply by jgonzalezalbisu** [reply]
I've done that and my ticket was closed suggesting me that I should post a thread in an existing question or post a new one. Seems like a never ending circle.

**Comment by jgonzalezalbisu** [comment]
Same issue here https://github.com/orgs/community/discussions/53695
